Breakout CSS and change it to be opt-in

pgrippi opened the pull request:
- Changes all CSS to be opt-in
- Break the CSS into two files: structure and appearance. Structure should define the bare-minimum CSS needed such that only it can be included for the table to work correctly. Appearance should define a custom look and feel. With that in mind, I'm open to moving definitions around as some could fall into either category.

One other change in here is the addition of the `ember-truth-helpers` dependency. In the [table-columns.hbs:18](https://github.com/cball/justa-table/blob/master/addon/templates/components/table-columns.hbs#L18) template the `eq` helper is used, without ember-truth-helpers it breaks the app.

Should close #9 if this is what you had in mind

This was made to fix a bug where I was forced to use `!important` in my CSS to change the background color of a row because it was [set by the CSS](https://github.com/cball/justa-table/blob/master/addon/styles/_justa-table.scss#L61) in this project.

Commit:
Breakout CSS, CSS is no longer included by default. Added missing dependency on ember-truth-helpers

cball commented:
@pgrippi thanks! this is exactly what I wanted to do.

cball commented on README.md:
```diff
@@ -2,11 +2,24 @@
-## Usage
+## Including In An Ember Application
+
+Here is the simplest way to get started with ember-modal-dialog:
```
should probably change the end of this line. :smile:

pgrippi replied:
oh crap

cball commented:
One other minor thing, can you name the `scss` files with underscores since they are meant to be imported into another stylesheet?

Commit:
Update the readme to be less copy-and-paste, add underscores to the scss file names as they are partials

pgrippi commented:
> can you name the `scss` files with underscores

I've updated the file names to start with an underscore, which is what I think you were saying. Did you want me to replace the hyphens with underscores too?

Also I fixed the README, I think I got a bit overzealous with my copy-and-pasting...

cball commented:
Sorry, pre :coffee: comment. prefix with underscores is what I was saying.
